translation-web/src/main/java/eu/europeana/api/translation/web/TranslationController.java

```diff
@@ -33,7 +35,7 @@ public TranslationController(TranslationWebService translationService) {
   @Tag(description = "Translation", name = "translate")
   @PostMapping(value = {"/translate"},
       produces = {HttpHeaders.CONTENT_TYPE_JSON_UTF8, MediaType.APPLICATION_JSON_VALUE})
-  public ResponseEntity<String> translate(@RequestBody TranslationRequest translRequest,
+  public ResponseEntity<String> translate(@ValidJson(uri = TranslationSchemaLocation.JSON_SCHEMA_URI, nested = TranslationSchemaLocation.JSON_SCHEMA_NESTED) TranslationRequest translRequest,
```

Comment:
I think you need two nested json schema one for detect and one for translate. here we should use the translate one right?